test/test.c

```diff
@@ -32121,6 +32121,16 @@ void test_KeyValueWatch(void)
     natsThread_Join(t);
     natsThread_Destroy(t);
     kvWatcher_Destroy(w);
+
```

Comment:
nit: As best practice, I would set `w` to `NULL` since you are going to recreate below. But since if it is fails to be created we fail (testCond() would exit), then there is no risk of "double free" when calling `kvWatcher_Destroy()` again...